### PR TITLE
Add EIP: Preimage retention

### PR DESCRIPTION
**Pull request by gballet**

### CODE REVIEW

File: EIPS/eip-6873.md [hunk 1]
```diff
@@ -0,0 +1,59 @@
+---
```
Comment (abcoathup) [inline EIPS/eip-6873.md:1]:
```suggestion
---
eip: 6873
```
Assigning PR number as EIP number

File: EIPS/eip-6873.md [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+title: Preimage retention in the fork preceding the Verge
```
Comment (abcoathup) [inline EIPS/eip-6873.md:2]:
```suggestion
title: Preimage retention
```
Short title

File: EIPS/eip-6873.md [hunk 3]
```diff
@@ -0,0 +1,59 @@
...
+description: Execution clients must retain the preimages of addresses and slots accessed between the fork preceding the verge, and the verge itself.
+author: Guillaume Ballet (@gballet)
+discussions-to: <URL>
```
Comment (SamWilsn) [inline EIPS/eip-6873.md:5]:
```suggestion
discussions-to: https://ethereum-magicians.org/t/eip-6873-preimage-retention-in-the-fork-preceding-the-verge/15830
```

File: EIPS/eip-6873.md [hunk 4]
```diff
@@ -0,0 +1,59 @@
...
+
+To guarantee a timely availability of all preimages around the verkle transition time, each node is therefore responsible for updating the list of preimages between the fork preceding the Verge, and the Verge itself.
+
+## Backwards Compatibility
+
+No backward compatibility issues found.
+
+## Test Cases
+
+TODO
```
Comment (SamWilsn) [inline EIPS/eip-6873.md:47]:
```suggestion
<!--

## Test Cases

TODO

-->
```

**Review by SamWilsn — APPROVED** [review]

**Pull request merged**